src/features/TrendingMovies/components/ImageSlider.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import ScheduleIcon from '@mui/icons-material/Schedule';
+
+interface Movie {
```

Comment:
Let's create a new folder near 'components' folder and call it 'types' and move this interface there